test-suite/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
Is this package in a separate workspace than the top-level WinterJS code? I'm not seeing anything that would put it in a separate workspace.

Reply:
No it's not in a separate workspace but rather it's own package

Reply:
NVM, the root `Cargo.toml` isn't a workspace.